consumer/consumer.go

```diff
@@ -78,6 +80,18 @@ func (c *Consumer) setup() {
 			return clusterConsumer(cons), err
 		}
 	}
+
+	if c.retryStrategy == nil {
+		// Note: the logic in handleMsg assumes that
+		// this does not terminate; be aware of that when changing
+		// this strategy.
+		c.retryStrategy = retry.Exponential{
+			Initial:  time.Millisecond,
+			Factor:   2,
+			MaxDelay: 5 * time.Second,
```

Comment:
It should be `MaxDelay: c.MaxRetryInterval,`, isn't it?

Reply:
yup, good catch!